ts/components/dialog/user-settings/pages/PreferencesSettingsPage.tsx

```diff
@@ -38,6 +39,19 @@ async function toggleStartInTray() {
   }
 }
 
+async function toggleAutoStart() {
+  try {
+    const currentSettings = await window.getAutoStartEnabled();
+    const newValue = !currentSettings;
+
+    // make sure to write it here too, as this is the value used on the UI to mark the toggle as true/false
+    await window.setSettingValue(SettingsKey.settingsAutoStart, newValue);
+    await window.setAutoStartEnabled(newValue);
```

Comment:
should we make those 2 calls only one, and call one from the other? I feel like we are going to forget at some point.

I guess we should keep `window.setAutoStartEnabled` here, and call 
`window.setSettingValue` in it, once we get that the change was made (and actually maybe do a get, instead of assuming the change worked?

Not sure if that's possible